### PR TITLE
bugfix(selectlist): selected value disappears

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

change value attribute's value to selected instead of default value

## Why

Selected value is changing back to default value

## Issue

NA

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: src/components/basic/SelectList/index.tsx [hunk 1]
```diff
@@ -60,6 +61,7 @@ export const SelectList = ({
   onChangeItem,
 }: SelectListProps) => {
   const selectHeight = popperHeight ? `${popperHeight}px` : 'auto'
+  const [selected, setSelected] = useState<any>({})
```
Comment (oyo) [inline src/components/basic/SelectList/index.tsx:64]: please stop using `any`- if necessary create a proper type and export it. If there was `any` before then we take the time to clean up dirty coding.

**Review by oyo — CHANGES_REQUESTED** [review]
any

File: src/components/basic/SelectList/index.tsx (hunk 1 above)
Comment (manojava-gk) [reply src/components/basic/SelectList/index.tsx:64]: this selected can only be `any`. We are not sure about the `items` type in this component.

**Review by oyo — APPROVED** [review]
The SelectList has a too generic design allowing `any` as items - we approve this for now and handle redesign in a separate ticket.

**Pull request merged**